portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,44 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <div id="root">
+          <div class="header-section">
+              <div class="column" style="background-image: url(/images/header/about.jpg);">
+                  <h3>01</h3>
+                  <h1>About</h1>
+              </div>
+              <div class="column" style="background-image: url(/images/header/projects.jpg);">
+                  <h3>02</h3>
+                  <h1>Projects</h1>
+              </div>
+              <div class="column" style="background-image: url(/images/header/gallery.jpg);">
+                  <h3>03</h3>
+                  <h1>Gallery</h1>
+              </div>
+              <div class="column" style="background-image: url(/images/header/me.jpg);">
+                  <h3>04</h3>
+                  <h1>Me</h1>
+              </div>
+          </div>
+          <div class="content-section">
+              <h1>About</h1>
+              <h1>Projects</h1>
+              <p>Developed a decentralized voting system over the Ethereum blockchain using smart contracts in Solidity. Microservices architecture built with Node.js, Docker, and IBM Cloud.</p>
+              <p>Created a Chrome extension for speed reading utilizing facial recognition to control speed, and play/stop the session. The extension was built with React and Javascript. The ML models use ML5 library to have them web based to ensure fast feedback.</p>
+              <p>Built a web page to facilitate communication between foreigners and foreign students to find roommates with similar lifestyles using web technologies such as HTML, CSS and Javascript and Reacts as a framework. </p>
+              <h1>Gallery</h1>
+              <p>I enjoy photographing architecture because it's the combination between math and art. It’s amazing how it reflects the lifestyle and ideologies of the society at the time of construction, so everytime you see a building you are looking directly at a time capsule. </p>
```

Comment:
nit: Some of these lines seem over 80 characters long. Try to keep the lines length under 80 characters per https://developers.google.com/style/html-formatting#line-length (Apply throughout the rest of the file).